packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -160,10 +160,6 @@ export type SliderProps = {
    * Allows props to be passed for slider-max
    */
   maxProps?: React.ComponentProps<'span'>;
-  /**
-   * Allows props to be passed for slider-rail
-   */
-  railProps?: React.ComponentProps<'div'>;
```

Comment:
I noticed that #1436 didn't add a changeset. So should we add a changeset now for the new props added in that PR? @mayank99 @gretanausedaite

Reply:
there should be a CSS changeset in #1436, and a react changeset in this PR.

this PR should also have no css/html changes if possible.